build/patches/Add-flag-to-always-view-the-desktop-site-for-all-websites.patch

```diff
@@ -471,6 +1123,38 @@ diff --git a/content/browser/frame_host/navigation_controller_android.h b/conten
    void ClearSslPreferences(
        JNIEnv* env,
        const base::android::JavaParamRef<jobject>& /* obj */);
+diff --git a/content/browser/renderer_host/render_process_host_impl.cc b/content/browser/renderer_host/render_process_host_impl.cc
+--- a/content/browser/renderer_host/render_process_host_impl.cc
++++ b/content/browser/renderer_host/render_process_host_impl.cc
+@@ -3503,6 +3503,7 @@ void RenderProcessHostImpl::PropagateBrowserCommandLineToRenderer(
+     switches::kIpcDumpDirectory,
+     switches::kIpcFuzzerTestcase,
+ #endif
++    switches::kDesktopModeViewportMetaEnabled,
+   };
+   renderer_cmd->CopySwitchesFrom(browser_cmd, kSwitchNames,
+                                  base::size(kSwitchNames));
+diff --git a/content/browser/web_contents/web_contents_impl.cc b/content/browser/web_contents/web_contents_impl.cc
+--- a/content/browser/web_contents/web_contents_impl.cc
++++ b/content/browser/web_contents/web_contents_impl.cc
+@@ -37,6 +37,7 @@
+ #include "base/strings/string_util.h"
+ #include "base/strings/utf_string_conversions.h"
+ #include "base/system/sys_info.h"
++#include "base/base_switches.h"
+ #include "base/threading/thread_task_runner_handle.h"
+ #include "base/time/time.h"
+ #include "base/trace_event/trace_event.h"
+@@ -2319,7 +2320,8 @@ const WebPreferences WebContentsImpl::ComputeWebPreferences() {
+ 
+   prefs.viewport_enabled = command_line.HasSwitch(switches::kEnableViewport);
+ 
+-  if (IsOverridingUserAgent())
++  if (IsOverridingUserAgent() && 
++      !command_line.HasSwitch(switches::kDesktopModeViewportMetaEnabled))
+     prefs.viewport_meta_enabled = false;
```

Comment:
>> added possibility to reactivate the metatag view for desktop mode, so that users can choose to use the flag in the hamburger menu to navigate with a custom useragent leaving the standard navigation unchanged

> I do not understand this part; what does it do?

related code is here.
in desktop mode the viewport tag (ex: `<meta name="viewport" content="width=device-width, initial-scale=1.0">`) is ignored by blink, so the css declared for the desktop are used